components/list/list.js

```diff
@@ -32,33 +42,39 @@ function List(props) {
 }
 
 List.defaultProps = {
-  hideBullets: false,
   align: 'vertical',
+  hideBullets: false,
+  renderEmptyItems: true,
```

Comment:
any case when it should be `true` by default?

Reply:
Not sure that there is any. I did that way just to save existing functionality.